### PR TITLE
Add -Xmx256m to JLM test processes, decrease thread limit in server load

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
resolves https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/249
igned-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Background: We need to re-calibrate the jlm tests and add -xmx256m to the sub-processes. We also need to change the number of threads the server is using from 200 to 30 - so that less number of threads are used. (details: https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/249)

**Comment by Mesbah-Alam** [comment]
(Grinder link shared with reviewer on slack)

File: openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmGCMXBeanNotifier.java [hunk 1]
```diff
@@ -108,7 +110,8 @@ public void execute(StfCoreExtension test) throws StfException {
 		
 		// Process definition for the monitoring client JVM that connects with the server			
         JavaProcessDefinition client2JavaInvocation =  test.createJavaProcessDefinition()
-		    .addProjectToClasspath("openjdk.test.jlm")
+    		.addJvmOption("-Xmx256m")
+        	.addProjectToClasspath("openjdk.test.jlm")
```
Comment (renfeiw) [inline openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmGCMXBeanNotifier.java]: I think you have accidentally replaced the tab wth space here.

File: openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmRemoteMemoryNoAuth.java [hunk 2]
```diff
@@ -121,7 +123,8 @@ public void execute(StfCoreExtension test) throws StfException {
 		dumpFile = resultsDir.childFile("javacore_mem_server.%Y%m%d.%H%M%S.%pid.%seq.txt,filter=java.lang.IllegalArgumentException");
 		
         JavaProcessDefinition clientJavaInvocationServer =  test.createJavaProcessDefinition()
-		    .addJvmOptionIfIBMJava("-Xdump:java:events=throw,file=" + dumpFile.getSpec())
+    		.addJvmOption("-Xmx256m")
+        	.addJvmOptionIfIBMJava("-Xdump:java:events=throw,file=" + dumpFile.getSpec())
```
Comment (renfeiw) [inline openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmRemoteMemoryNoAuth.java]: Same with comment in TestJlmGCMXBeanNotifier.java, you may want to replace space back with tab.

Comment (Mesbah-Alam) [reply openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmRemoteMemoryNoAuth.java]: Thanks! Corrected it.

File: openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmGCMXBeanNotifier.java (hunk 1 above)
Comment (Mesbah-Alam) [reply openjdk.test.jlm/src/test.jlm/net/adoptopenjdk/test/jlm/TestJlmGCMXBeanNotifier.java]: Thanks! Corrected it.

**Pull request merged**